grafana/Dockerfile

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
We don't really need this new line :-)

Reply:
Aww, I liked the line. Even though it was by accident it was one of my favourites. Removed :-)